v1 support date in "REST API" articles (#3307)
MicrosoftDocs/edge-developer: microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

<!-- ------------------------------ -->
#### Versions of the Update REST API

> As of September 6, 2024, both v1.1 and v1 of this Update REST API are supported.  Later, v1 will no longer be supported.  The date for ending v1 support is December 31, 2024.

Reviewer: ```suggestion
As of September 6, 2024, both v1.1 and v1 of this Update REST API are supported.  Support for v1 will end on December 31, 2024.
```
